ngraph/frontend/onnx_import/src/editor/editor.cpp

```diff
@@ -232,3 +302,31 @@ void onnx_import::ONNXModelEditor::set_input_shapes(
         }
     }
 }
+
+void onnx_import::ONNXModelEditor::set_input_values(
+    const std::map<std::string, std::shared_ptr<ngraph::op::Constant>>& input_values)
+{
+    auto* onnx_graph = m_pimpl->m_model_proto.mutable_graph();
+
+    for (const auto& input : input_values)
+    {
+        auto& name = input.first;
+        auto& values = input.second;
+
+        auto input_desc = find_graph_input(*onnx_graph, input.first);
+        auto initializer_desc = find_graph_initializer(*onnx_graph, name);
+
+        if (!initializer_desc && !input_desc)
+        {
+            NGRAPH_INFO << "There is no input nor initializer named '" << name
```

Comment:
Shouldn't it be logged with NGRAPH_WARN?

Reply:
Adding initializer into an invalid model might (assuming that it's the only invalidity) fix the model. Such operation is rather intentional, so I think WARN would be inadequate.